content/kubernetes/re-databases/set-up-ingress-controller.md

```diff
@@ -88,9 +88,12 @@ Install one of the supported ingress controllers:
         http:
           paths:
           - path: /
+            pathType: Prefix
             backend:
-              serviceName: <db-name>
-              servicePort: <db-port>
+              service:
+                name: <db-name>
+                port:
+                  number: <db-port>
```

Comment:
`port.number` is valid, but we could also instruct users to set `port.name: redis` instead, which is the default port name used in the REDB service. This should work out of the box and spare the user from disovering the REDB service port.

Reply:
Testing is always a good idea, especially when feeling (too) comfortable.